p-token/src/entrypoint-runtime-verification.rs

```diff
@@ -750,18 +750,18 @@ fn inner_test_validate_owner(
             if signers_count < multisig.m as usize {
                 assert_eq!(result, Err(ProgramError::MissingRequiredSignature));
                 return result;
-            } else {
-                return result;
             }
+
+            return Ok(());
```

Comment:
This is the most important change, avoids erroring out at the call site.
https://github.com/runtimeverification/solana-token/blob/222086465cecda35736c17783179f67d0b3f6034/p-token/src/entrypoint-runtime-verification.rs#L1393-L1401